README.md

```diff
@@ -45,6 +60,28 @@ risk candidates for security issues. Therefore the security team puts
 restrictions on the introduction of new cron jobs or changes to existing cron
 jobs.
 
+### Device Files
+
+The file `device-files-whitelist.json` contains whitelisting entries for
+device file packaged in RPMs. Device files in RPMs should be an unusual
+event. Since device files with bad permissions may allow unprivileged users to
+access sensitive system devices it is important to restrict packaging of this
+type of files. A metadata whitelisting is used for whitelist any occurences of
+device files in packages.
+
+### World Writable Files
+
+There shouldn't be any files packaged that are world-writable for anybody. A
```

Comment:
-" for anybody". world-writeable implies this

Reply:
True, good catch :)